app/views/people/index.html.erb

```diff
@@ -0,0 +1,28 @@
+<h1>Task Owner List</h1>
+
+<table>
+  <tr>
+    <th>
+      Name
+    </th>
+    <th>
+      Incomplete Tasks
+    </th>
+  </tr>
+  <tbody>
+    <% @all_people.each do |person| %>
+    <tr>
+      <td>
+        <%= link_to "#{person.name}", "/people/#{person.id}" %>
+      </td>
+      <td>
+        <% if Task.where("person_id = #{person.id}").present? %>
+          <%= Task.where("person_id = #{person.id}").where("completed_date IS NULL ").count %>
```

Comment:
Because of the `has_many` association on `Person` you can simplify these `where` calls by just saying `person.tasks`:

```ruby
if person.tasks.present?
  person.tasks.where(...).count
```

Do you think it's helpful to have "" versus "0" in the UI depending on whether they have any tasks at all? I as because this makes two database calls and it would make things (inconsequentially) faster to just make the one call.